pipelines/qc/Snakefile

```diff
@@ -1,32 +1,69 @@
 import os
 import numpy as np
 import pandas as pd
-configfile: "config.yml"
 
-samples = pd.read_csv(config["manifest"], index_col=0)
-script_dir = config["scripts_loc"] 
+sampleset = os.path.split(os.getcwd())[1]
+if os.path.isfile("configuration.yml"):
+  os.remove("configuration.yml")
 
-rule all:
-  input: 
-    contam=expand("output/{sample}.contamination.csv", sample=samples.index.tolist()),
-    alignm=expand("output/{sample}.callstats.csv", sample=samples.index.tolist())
+# read_fofn
+fofn_path = os.path.join(config["vector_ops"], "tracking", sampleset, "agam.fofn.tsv")
+fofn = pd.read_table(fofn_path)
+samples = fofn['sample'].unique()
+
+# set script dir
+script_dir = "../../scripts"
```

Comment:
According to snakemake conventions (verbosity > compactness), we should either move this to config or do not use the variable at all. Given that scripts location is relative to Snakefile and we are not planning to move it for now, second option seems plausible